scripts/prepare_sector_network.py

```diff
@@ -1521,6 +1521,22 @@ def add_heat(n, costs):
                 capital_cost=costs.at[key, 'efficiency'] * costs.at[key, 'fixed'],
                 lifetime=costs.at[key, 'lifetime']
             )
+            
+            if options["biomass_boiler"] and name not in ["urban central"]:
+                #TODO: Add surcharge for pellets
+                n.madd("Link",
+                    nodes[name] + f" {name} biomass boiler",
+                    p_nom_extendable=True,
+                    bus0=spatial.biomass.df.loc[nodes[name], "nodes"].values,
+                    bus1=nodes[name] + f" {name} heat",
+                    bus2="co2 atmosphere",
+                    carrier=name + " biomass boiler",
+                    efficiency=costs.at['biomass boiler', 'efficiency'],
+                    efficiency2=costs.at['solid biomass', 'CO2 intensity']-costs.at['solid biomass', 'CO2 intensity'],
```

Comment:
the biomass boilers do not emit any CO2 are they? If not, one can remove the connection to` bus2` and `efficiency2 `when adding them

Reply:
Well, no net emissions as long as biomass is assumed to be carbon neutral. So yes this connection is technically not necessary (but maybe good to keep for consistency with other biomass techs?). I think both are ok, you decide!

Reply:
Alright, I am going to remove it for now to be consistent with how we implement the [biomass CHPs](https://github.com/PyPSA/pypsa-eur-sec/blob/654ef435deb34ceac98902a09e7a3db2223da622/scripts/prepare_sector_network.py#L1831-L1843) for example

Reply:
one more question @millingermarkus the costs and efficiencies for the biomass boilers should they be `solid biomass boiler steam` in the technology-data or did I missed something when pulling your changes into the main code?

Reply:
No, that tech was missing in the main technology data, I made a pull request now: https://github.com/PyPSA/technology-data/pull/61